opus/application/static_media/js/browse.js

```diff
@@ -1260,70 +1325,72 @@ var o_browse = {
         if (!$(selector).data("infiniteScroll")) {
             $(selector).infiniteScroll({
                 path: function() {
-                    console.log(`=== PATH ON ${selector} ===`);
                     let startObs = opus.prefs[startObsLabel];
+                    let customizedLimitNum;
+                    let lastObs = $(`${tab} .thumbnail-container`).last().data("obs");
 
-                    console.log(`${selector} in path - startObs: ${startObs}`);
                     let infiniteScrollData = $(selector).data("infiniteScroll");
                     if (infiniteScrollData !== undefined && infiniteScrollData.options.loadPrevPage === true) {
-                        // Direction: scroll up
-                        console.log(`loadPrevPage = true`);
-                        infiniteScrollData.options.loadPrevPage = false;
+                        // Direction: scroll up, we prefetch 1 * o_browse.getLimit() items
+                        if (startObs !== 1) {
+                            let originalStartObs = $(`${tab} .thumbnail-container`).first().data("obs");
```

Comment:
We shouldn't call these "StartObs". StartObs is a specific thing - the slug that indicates the top left corner. Just call it "ObsNum".